### PR TITLE
Feature branch/jenn

### PR DESCRIPTION
**Pull request by jennxu23**
# Jennifer Xu View
## :recycle: Current situation & Problem
I didn't have a page in the CS342 app.

## :bulb: Proposed solution
I added my personal page, which includes a picture of me, my social media links, a short bio, and a timeline of recent professional milestones.

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

I used the following youtube video as inspiration for my Timeline View: [SwiftUI 3.0 - Task Management App UI - Complex UI - MVVM - Xcode 13 - SwiftUI Tutorials
](https://www.youtube.com/watch?v=nKHrsrmA4lM&ab_channel=Kavsoft)

I pulled the LinkedIn Icon from their brand package: [LinkedIn Brand Guidelines](https://brand.linkedin.com/downloads)

I pulled the GitHub Icon from their brand package: [GitHub Logos and Usage](https://github.com/logos)

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

I added a few UI tests. There is a unit test for each of the social media buttons. There is also a basic unit test ensuring that the section titles properly renders.

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CS342MemberViews/Sources/JenniferXu/JenniferXu.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+// SPDX-License-Identifier: MIT
+//
+// Jennifer Xu Member View. Modeled partially after Paul Schmiedmayer's Member View.
+//
+// Cited:
+// GitHub Logo: GitHub Logos and Usage - https://github.com/logos
+// LinkedIn Logo: LinkedIn Brand Guidelines - https://brand.linkedin.com/downloads
+//
+//  Created by Jennifer Xu on 1/19/23.
+//
```
Comment (jennxu23) [inline CS342MemberViews/Sources/JenniferXu/JenniferXu.swift]: @PSchmiedmayer @vishnuravi I'm not sure if I did the licensing correctly. Any advice is appreciated!

File: CS342MemberViews/Sources/JenniferXu/JenniferXu.swift [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                       .font(.title2.bold())
+                   Text("B.S Biomedical Computation\nM.S Computer Science")
+                       .font(.title3)
+               }
+               
+               //Social Media links
+               HStack(){
+                   SocialButton(social: "LI-In-Bug", url:"https://www.linkedin.com/in/jenniferxu01/", label: "linkedin")
+                   SocialButton(social: "github-mark", url:"https://github.com/jennxu23", label:"github" )
+                   SocialButton(url:"https://jennxu.netlify.app/", label: "personal")
```
Comment (jennxu23) [inline CS342MemberViews/Sources/JenniferXu/JenniferXu.swift]: I'm not sure how to handle the licensing of these images. I pulled the icons from the official branding packages of LinkedIn and GitHub.
